drv/tdma_client.h

```diff
@@ -0,0 +1,106 @@
+#ifndef __TDMA_CLIENT_H
+#define __TDMA_CLIENT_H
+
+/**
+ * @defgroup    drv_tdma_client      TDMA client radio driver
+ * @ingroup     drv
+ * @brief       Driver for Time-Division-Multiple-Access fot the DotBot radio
+ *
+ * @{
+ * @file
+ * @author Said Alvarado-Marin <said-alexander.alvarado-marin@inria.fr>
+ * @copyright Inria, 2024-now
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <nrf.h>
+#include "gpio.h"
+#include "radio.h"
+#include "protocol.h"
+
+//=========================== defines ==========================================
+
+///< TDMA internal registrarion state
```

Comment:
```suggestion
/// TDMA internal registrarion state
```

The `<` is used when the doxygen comment is place on the same line as the thing it document, at the end. When the doxygen comment is placed above, use `///`. Sorry to only notice that now.